underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java

```diff
@@ -653,12 +654,7 @@ private String stripFolderSuffixIfPresent(String key) {
    * @return the key without the gcs bucket prefix
    */
   private String stripPrefixIfPresent(String key) {
-    if (key.startsWith(mBucketPrefix)) {
-      return key.substring(mBucketPrefix.length());
-    }
-    if (key.startsWith(PATH_SEPARATOR)) {
-      return key.substring(PATH_SEPARATOR.length());
-    }
-    return key;
+    key = CommonUtils.stripPrefixIfPresent(key, mBucketPrefix);
+    return CommonUtils.stripPrefixIfPresent(key, PATH_SEPARATOR);
```

Comment:
This does not provide the same behavior as before, ie. `prefix:///file` will be parsed differently.